public/app/plugins/datasource/prometheus/datasource.ts

```diff
@@ -483,10 +474,6 @@ export class PrometheusDatasource
     return processedTargets;
   }
 
-  intepolateStringHelper = (query: PromQuery): string => {
-    return this.interpolateString(query.expr);
-  };
-
```

Comment:
@gtk-grafana I think that was used for testing and then forgotten. It wasn't being used anywhere so I removed it.